### ISSUE TITLE
How to use the allowed_elements parameter to allow Inline HTML [Element: a]

### ISSUE THREAD

**Issue by serenity-hub** [DavidAnson/vscode-markdownlint#152]
# Rule triggered

"MD033/no-inline-html: Inline HTML [Element: a]"

*(I understand the reason.)*

## Looking for enlightenment on

**Easy way** to disable rule in Visual Studio Code user/workspace `settings.json`

---

### What I have done

I added code in `settings.json` but it doesn't work, **I suspect my code is incorrect**.

    "markdownlint.config": {
        "default": true,
        "MD033": {
          "allowed_elements": [ "a" ]
        }
      }

---

### Other things I've tried

- I've read Configure at [markdownlint - Visual Studio Marketplace](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint "markdownlint - Visual Studio Marketplace")
- Web searched 
- GitHub searched.
- Tried random suggestions and code.
*(Though I'm not looking to create additional files and to try to resolve this through `settings.json`, I have tried instructions related to create `.markdownlint.json`, but I have failed. Possible reason, I don't understand code.)*

---
Any guidance is appreciated.

About
Version: 1.52.1
Electron: 9.3.5
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Linux x64 5.8.0-41-generic snap

**Comment by DavidAnson** [comment]
Here's an example workspace that disables `MD033` for `a` elements in VS Code with the `markdownlint` extension:

```bash
user@host issue152 % cat file.md
# Issue 152

<a href="https://example.com">Inline HTML</a>
user@host issue152 % cat .vscode/settings.json
{
    "markdownlint.config": {
        "MD033": {
            "allowed_elements": [ "a" ]
        }
    }
}
user@host issue152 % code .
```

**Comment by serenity-hub** [comment]
#  [FIXED] Thank you very much @DavidAnson 

Here is what I was doing wrong.  
Because I wanted to separate my setting from the rest, by copy/paste in `settings.json` I was breaking JSON curly bracket rules like this

    {
        "editor.someSetting": true,
        "editor.anotherSetting": false,
        "editor.onemoreSetting": true,
    }
    
    {
        "markdownlint.config": {
            "MD033": {
                "allowed_elements": [ "a" ]
            }
        }
    }

Instead of the **correct way** like this and leaving space for visual separation.

    {
        "editor.someSetting": true,
        "editor.anotherSetting": false,
        "editor.onemoreSetting": true,
        
        "markdownlint.config": {
            "MD033": {
                "allowed_elements": [ "a" ]
            }
        }
    }
